OrbitBase/ThreadPool.cpp

```diff
@@ -67,6 +80,21 @@ void ThreadPoolImpl::Schedule(std::unique_ptr<Action> action) {
       worker_threads_.size() < thread_pool_max_size_) {
     CreateWorker();
   }
+
+  CleanupFinishedThreads();
```

Comment:
It seems CleanupFinishedThreads() is called either when an action is scheduled or when shutting down (unless I'm missing something).  Doesn't it mean that we could have a bunch of threads lying around if there is no Schedule call?

Reply:
It does.. but this situation is unlikely. Someone at some point is going to call schedule. And having some idle and waiting for join threads is not much of an overhead.

Reply:
Fair enough.